Mastodon/Protocol/Provider/DataSourceProvider+StatusTableViewCellDelegate.swift

```diff
@@ -709,21 +682,21 @@ extension StatusTableViewCellDelegate where Self: DataSourceProvider & AuthConte
                 return
             }
             switch item {
-            case .status(let status):
-                await DataSourceFacade.coordinateToStatusThreadScene(
-                    provider: self,
-                    target: .status,    // remove reblog wrapper
-                    status: status
-                )
-            case .user(let user):
-                await DataSourceFacade.coordinateToProfileScene(
-                    provider: self,
-                    user: user
-                )
-            case .notification:
-                assertionFailure("TODO")
-            default:
-                assertionFailure("TODO")
+                case .status(let status):
```

Comment:
Same as my comment regarding indentations above, re-indenting in Xcode it will produce changes here 🤔